data-lineage/src/main/java/com/wipro/ats/bdre/lineage/api/PersistenceUnit.java

```diff
@@ -37,7 +38,7 @@
 	private UniqueList<LineageNodeInfo> lineageTableNodes = new UniqueList<LineageNodeInfo>();
 	private UniqueList<LineageNodeInfo> lineageColumnNodes = new UniqueList<LineageNodeInfo>();
 	private List<LineageNodeInfo> lineageFunctionNodes = new UniqueList<LineageNodeInfo>();
-	private LineageQueryInfo lineageQueryInfo = null;
+	//private LineageQueryInfo lineageQueryInfo = null;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3ACommentedOutCodeLine)